src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/CSharpFormattingPassBase.cs

```diff
@@ -56,6 +56,9 @@ protected async Task<List<TextChange>> AdjustIndentationAsync(FormattingContext
             foreach (var mapping in context.CodeDocument.GetCSharpDocument().SourceMappings)
             {
                 var mappingSpan = new TextSpan(mapping.OriginalSpan.AbsoluteIndex, mapping.OriginalSpan.Length);
+#if DEBUG
+                var spanText = context.SourceText.GetSubText(mappingSpan).ToString();
```

Comment:
Is this leftover code from debugging?

Reply:
Yes, but its deliberate, to make future debugging easy too.